src/schematools/contrib/django/models.py

```diff
@@ -549,26 +549,28 @@ def __str__(self):
         return self.name
 
 
-class LooseRelationField(models.CharField):
-    def __init__(self, *args, relation, to_field=None, **kwargs):
-        kwargs.setdefault("max_length", 254)
-        super().__init__(*args, **kwargs)
-        self.relation = relation
-        self.to_field = to_field
+class LooseRelationField(models.ForeignKey):
+    """A relation that points to one part of the composite key.
 
-    def deconstruct(self):
-        name, path, args, kwargs = super().deconstruct()
-        del kwargs["max_length"]
-        kwargs["relation"] = self.relation
-        if self.to_field:
-            kwargs["to_field"] = self.to_field
-        return name, path, args, kwargs
+    This only points to the first field of the relation, and the second field
+    (e.g. volgnummer/beginGeldigheid) is determined at runtime using query filtering.
+    Without such filter, traversing the relation will produce multiple results.
+
+    This setup is typically used for temporal relationships, where the foreign table uses compound
+    keys as identifier. This field type references the first field of that compound relationship.
+
+    At construction, the "to_field" will be set pointing to the correct identifier field.
+    """
+
+    # Disable the unique check made against the target field.
+    # When it's part of a composite key, this can't be unique by itself.
```

Comment:
```suggestion
    # When it's part of a composite key, this can't always be unique by itself.
```

(It can be unique, it just doesn't have to be.)